spec.html

```diff
@@ -34329,7 +34329,10 @@ <h1>WeakMap.prototype.delete ( _key_ )</h1>
           1. If Type(_M_) is not Object, throw a *TypeError* exception.
           1. If _M_ does not have a [[WeakMapData]] internal slot, throw a *TypeError* exception.
           1. Let _entries_ be the List that is _M_.[[WeakMapData]].
-          1. If Type(_key_) is not Object, return *false*.
+          1. If Type(_key_) is Symbol, then,
+            1. For each element _e_ of the GlobalSymbolRegistry List, do,
+              1. If SameValue(_e_.[[Symbol]], _value_) is true, return *false*.
```

Comment:
As noted in https://github.com/tc39/ecma262/issues/1194#issuecomment-393143921 , I object to this increase in complexity and cognitive burden.